pkg/internal/cyberark/servicediscovery/testdata/README.md

```diff
@@ -0,0 +1,3 @@
+# Test data for CyberArk Discovery
+
+All data in this folder is derived from an unauthenticated endpoint accessible from the public internet.
```

Comment:
I'd like to know how to recreate or update this data.
I tried using curl , but got an empty response.

```
$ curl https://platform-discovery.integration-cyberark.cloud/api/v2/services/subdomain/venafi-test; echo
{}
```

Reply:
That tenant was decommissioned and replaced with a new one I'm using to test the Identity client. There's a different example tenant which can be used for getting this kind of data: `curl -v https://platform-discovery.integration-cyberark.cloud/api/v2/services/subdomain/beyonceknowles`

I didn't want to add the old one or the one above to that README because I don't know if they're intended to be long lived, so my instinct is to avoid referring to either in code right now!